File: RestApi/Types/Dataset.py
class Dataset:
    def __init__(self, dataset_id, label="", color="rgb(63,81,181)", border_width=1.0, fill=False, tension=1.0):
        self.dataset_id = dataset_id
        self.label = label
        self.color = color
        self.border_width = border_width
        self.fill = fill
        self.tension = tension

    def to_dict(self):
        return dict(self.__dict__)

File: frontend/src/app/Types/Dataset.ts
export class Dataset {
  constructor(
    public dataset_id: string,
    public label: string = '',
    public color: string = 'rgb(63, 81, 181)',
    public border_width: Number = 1.0,
    public fill: boolean = false,
    public tension: number = 1.0
  ) {}
}
